projects/dashboards/esg-indicator-service/src/app/quota-log/quota-log.component.ts

```diff
@@ -39,7 +39,9 @@ export class QuotaLogComponent implements AfterViewInit {
     }
 
     ngAfterViewInit(): void {
-        this.paginator.page.pipe(tap(() => this.loadQuotaLogsPage())).subscribe();
+        this.paginator()
+            .page.pipe(tap(() => this.loadQuotaLogsPage()))
```

Comment:
can we useEffect here?

Reply:
EventEmitter is an async stream, so right now not necessarily.